src/core/core.scale.js

```diff
@@ -1309,6 +1319,8 @@ export default class Scale extends Element {
       align = 'left';
     } else if (ticks.align === 'end') {
       align = 'right';
+    } else if (ticks.align === 'left-right') {
+      align = 'left-right';
```

Comment:
In general we've tried to keep the public facing options direction agnostic (i.e. 'start' instead of 'left'). Perhaps this could be named `'auto'`?

Reply:
I like suggestion from LeeLenaleee and renamed this alligment type to "inner"